package.json

```diff
@@ -62,20 +58,24 @@
     "eslint-plugin-standard": "4.0.0",
     "eth-gas-reporter": "0.1.12",
     "eth-sig-util": "^2.1.1",
-    "ethereumjs-wallet": "0.6.2",
+    "ethereumjs-wallet": "0.6.3",
     "ethers": "4.0.13",
-    "moonwalker": "github:atvanguard/moonwalker#4c4592e2d6c1744fbcd9452cf0f00e9beb236b21",
+    "moonwalker": "github:gatsbyz/moonwalker",
```

Comment:
If we're going to do something like this, it should be moved into the `maticnetwork` organization.